analytics/javaspark/src/main/java/mil/nga/giat/geowave/analytic/javaspark/sparksql/udf/GeomFunction.java

```diff
@@ -0,0 +1,28 @@
+package mil.nga.giat.geowave.analytic.javaspark.sparksql.udf;
+
+import org.apache.spark.sql.api.java.UDF2;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.geom.Geometry;
+
+import mil.nga.giat.geowave.analytic.javaspark.sparksql.util.GeomReader;
+
+public abstract class GeomFunction implements
+		UDF2<String, String, Boolean>
+{
+	private final static Logger LOGGER = LoggerFactory.getLogger(GeomFunction.class);
+	protected GeomReader geomReader = new GeomReader();
```

Comment:
this is why the reader must be serializable (I think marking this transient shouldn't require it to be serializable)